test/global-rate-limit.test.js

```diff
@@ -98,6 +99,48 @@ test('With text timeWindow', async t => {
   t.equal(res.headers['x-ratelimit-remaining'], 1)
 })
 
+test('When passing NaN to the timeWindow property then the timeWindow should be the default value - 60 seconds', async t => {
+  t.plan(5)
+
+  t.context.clock = FakeTimers.install()
+
+  const defaultTimeWindowInSeconds = 60
+
+  const fastify = Fastify()
+  fastify.register(rateLimit, { max: 1, timeWindow: NaN })
+
+  fastify.get('/', async (req, reply) => 'hello!')
+
+  let res
+
+  res = await fastify.inject('/')
+
+  t.equal(res.statusCode, 200)
+  t.equal(res.headers['x-ratelimit-reset'], defaultTimeWindowInSeconds)
+
+  res = await fastify.inject('/')
+
+  t.equal(res.statusCode, 429)
+
+  // Wait for almost 60s to make sure the time limit is right
+  t.context.clock.tick(55_000)
```

Comment:
```suggestion
  t.context.clock.tick(55000)
```
that's not supported under Node.js 10

Reply:
Thank you, I saw that in the CI my new commit don't use them